Tests/common_BioSQL.py

```diff
@@ -323,11 +323,8 @@ def test_server(self):
         del server["biosql-test"]
         del server["biosql-test2"]
         self.assertEqual(0, len(server))
-        try:
+        with self.assertRaises(KeyError):
```

Comment:
Looks good - If I recall correctly, ``self.assertRaises`` on the older versions of Python we supported when that test was written.